Gems/RecastNavigation/Code/Source/EditorComponents/EditorRecastNavigationPhysXProviderComponent.cpp

```diff
@@ -27,7 +27,28 @@ namespace RecastNavigation
                     "[Collects triangle geometry from PhysX scene for navigation mesh within the area defined by a shape component.]")
                     ->ClassElement(AZ::Edit::ClassElements::EditorData, "")
                     ->Attribute(AZ::Edit::Attributes::AppearsInAddComponentMenu, AZ_CRC("Game"))
-                    ->Attribute(AZ::Edit::Attributes::AutoExpand, true);
+                    ->Attribute(AZ::Edit::Attributes::AutoExpand, true)
+                    ->Attribute(AZ::Edit::Attributes::Visibility, AZ::Edit::PropertyVisibility::ShowChildrenOnly)
+                ;
+
+                editContext->Class<RecastNavigationPhysXProviderComponentController>(
+                    "RecastNavigationPhysXProviderComponentController", "")
+                    ->ClassElement(AZ::Edit::ClassElements::EditorData, "")
+                    ->Attribute(AZ::Edit::Attributes::AutoExpand, true)
+                    ->DataElement(AZ::Edit::UIHandlers::Default, &RecastNavigationPhysXProviderComponentController::m_config, "Configuration", "")
+                    ->Attribute(AZ::Edit::Attributes::Visibility, AZ::Edit::PropertyVisibility::ShowChildrenOnly)
+                    ;
+
+                editContext->Class<RecastNavigationPhysXProviderConfig>("Recast Navigation PhysX Provider Config",
+                    "[Navigation PhysX Provider configuration]")
+                    ->ClassElement(AZ::Edit::ClassElements::EditorData, "")
+                    ->Attribute(AZ::Edit::Attributes::AppearsInAddComponentMenu, AZ_CRC("Game"))
+                    ->Attribute(AZ::Edit::Attributes::AutoExpand, true)
+
+                    ->DataElement(AZ::Edit::UIHandlers::Default, &RecastNavigationPhysXProviderConfig::m_collisionGroupId, "Collision Group",
+                        "If set, only colliders from the specified collision group will be considered.")
+                    ->Attribute(AZ::Edit::Attributes::ChangeNotify, AZ::Edit::PropertyRefreshLevels::ValuesOnly)
```

Comment:
If no other values are changing or updating in the inspector as a result of this then I don't think you need to change notify here.

Reply:
Thanks, removed.